src/Plugin/Block/SimpleSearchFormBlock.php

```diff
@@ -85,7 +85,7 @@ public function blockForm($form, FormStateInterface $form_state) {
     $form['action_path'] = [
       '#type' => 'textfield',
       '#title' => $this->t('Path'),
-      '#description' => $this->t('The path to redirect to.'),
+      '#description' => $this->t('The path to redirect to. Should start with a slash.'),
```

Comment:
I bet there's existing element to input path, at least views and probably menu using it